.stickler.yml

```diff
@@ -0,0 +1,16 @@
+linters:
+  rubocop:
+    display_cop_names: true
+    fixer: true
+    
+files:
+  ignore:
+    - "bin/*"
+    - "db/*"
+    - "config/*"
+    - "Guardfile"
+    - "Rakefile"
+    
+fixers:
+  enable: true
```

Comment:
😔.

Reply:
i think was a VS.Code plugin, that change the stickler, i always fix the issues manually, is cheaper than therapy 😂

I will create a edge case if ruby detect incompatibilities with emoji.

Reply:
i solve the problem, could i continue with the next milestone ?

Reply:
Hi! Could you request a new code review.